test-cypress/integration/explore/similarity.spec.js

```diff
@@ -0,0 +1,257 @@
+import ImportSteps from '../../steps/import-steps';
+
+describe('Similarity screen validation', () => {
+
+    const INDEX_NAME = 'index-' + Date.now();
+    const FILE_TO_IMPORT = 'people.zip';
+    const SUCCESS_MESSAGE = 'Imported successfully';
+    const INDEX_CREATE_URL = '/similarity/index/create';
+    const BUILD_PARAM = ' -trainingcycles 4';
+    const STOP_WORD = 'stopword';
+    const LUCENE_ANALYZER = 'org.apache.lucene.analysis.de.GermanAnalyzer';
+
+    let repositoryId;
+
+    beforeEach(() => {
+        repositoryId = 'repo-' + Date.now();
+        cy.createRepository({id: repositoryId});
+
+        ImportSteps.visitServerImport(repositoryId);
+        ImportSteps.selectServerFile(FILE_TO_IMPORT)
+            .importServerFiles()
+            .verifyImportStatus(FILE_TO_IMPORT, SUCCESS_MESSAGE);
+        cy.presetRepositoryCookie(repositoryId);
```

Comment:
`visitServerImport` will select the repository manually. The other option is to move this before vising the import page and NOT providing `repositoryId` as param. 

Note: In #50 I've updated `visitServerImport` to internally use `presetRepositoryCookie`